### ISSUE TITLE
Bug: not work on Ubuntu, need to use /etc/default/locale

### ISSUE THREAD

**Issue by laurent-kling** [robertdebock/ansible-role-locale#1]
### Bug

On Ubuntu, the config file is different.

Centos: `/etc/locale.conf`

```
- name: configure locale.conf centos
  template:
    src: locale.conf.j2
    dest: /etc/locale.conf
  notify:
    - localectl set-locale
```

Ubuntu: `/etc/default/locale`
https://www.thomas-krenn.com/en/wiki/Configure_Locales_in_Ubuntu

```
- name: configure locale.conf ubuntu
  template:
    src: locale.conf.j2
    dest: /etc/default/locale
  notify:
    - localectl set-locale
```

**Comment by robertdebock** [comment]
Thanks @laurent-kling, I'll change this asap.

**Comment by robertdebock** [comment]
Should be fixed by the last release, 1.0.1. Thanks @laurent-kling!

1.0.1 should be available in 30 minutes or so, please verify if this now works as expected and feel free to reopen this issue, or open another one. It's greatly appreciated!
